Magento_Vault/templates/cards_list.phtml

```diff
@@ -12,15 +12,15 @@
                 <thead>
                     <tr>
                         <th scope="col" class="col card-number">
-                            <?php /* @noEscape */ echo __('Card Number') ?>
+                            <?php echo __('Card Number') ?>
```

Comment:
Replace `echo` with shorthand here and below

Reply:
done